bankroll/converter/modelconverter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, cast, List, Optional, Sequence, Tuple, Union
+
+try:
+    import pandas as pd
+except ImportError:
+    pass
```

Comment:
IIRC this still needs to set `pandas` to avoid an undefined symbol error if the name is ever referenced:
```suggestion
    pandas = None
```

Reply:
✅